doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -49,7 +49,7 @@ The control command is indicated in B<cmd> and any additional arguments in
 B<p1> and B<p2>.
 
 For B<cmd> = B<EVP_PKEY_CTRL_SET_MAC_KEY>, B<p1> is the length of the MAC key,
-and B<p2> is MAC key. This is used by Poly1305, HMAC and CMAC.
+and B<p2> is MAC key. This is used by Poly1305, SipHash, HMAC and CMAC.
```

Comment:
it was a comment on how good standard API's are.  not asking for more work on this.